### DISCUSSION TITLE
Commission/Requests for Apps not on Android

### DISCUSSION THREAD

**Discussion by ghost** [Arcticons-Team/Arcticons#1697] [Q&A] [answered]
Hi, I've started using Arcticons Dark for my desktop icons as well since most of my programs have Android counterparts, and it's worked well. 

Though sadly there are several that of course aren't on Android (Like LibreWolf, the FireFox Fork I use) and trying to request them through the app is a bit difficult as there isn't an Android app icon for the request to use. 

Is there any way I can request/commission icons for apps not already on Android? Thanks!

**Comment by Donnnno** [comment, marked as answer]
Hi!

Since, the app focuses on Android use, requests go through the app. But other icons are allowed too, like the Linux ones we're working on right now. 

I do priority requests via [Ko-Fi](https://ko-fi.com/donno_), you can check that out, but if that's not an option, maybe some of the other contributors are willing to help you out!
